docs/scenarios_authoring/Enrichers.md

```diff
@@ -41,6 +41,28 @@ DatabaseLookupEnricher is a specialized look-up component; it returns all column
 
 ![alt_text](img/databaseLookupEnricher.png "databaseLookup Enricher")
 
+### DecisionTable enricher
+
+This component allows the user to match input to a pre-defined result. Please be wary that this component works a bit different than a classical decision table([Decision tables in Wikipedia](https://en.wikipedia.org/wiki/Decision_table)). It consists of the fields:
+- Name: the name of the component.
+- Decision Table: a user defined table of type TabularTypedData, discussed below.
+- Match condition: a Boolean SpEL expression that matches input to table rows, discussed below.
+- Output: name of the output variable which will hold data generated by the component.
+- Description: user-defined description of the particular component. 
+
+The component has two main elements: _Decision Table_ which is a user-defined table of values and _Match condition_ - a SpEL expression which Nu will use to match the input to the table values.
+ 
+First we have to define a fixed-value decision table. We will use a simple car mechanic concept - a decision table that helps to define whether a car should be serviced. The blue columns represent our input values and the red one is our decision column.
+![alt_text](img/decisionTableValues.png "basic decision table")
+
+We now need to define a condition that will match our input data with the table rows. In our case the input is a tuple (engine\_status, controls\_status, car\_id). We want to match when the values are equal to the row values. The relevant expression is:
+![alt_text](img/decisionTableExpression.png "Expression matching input with rows.")
+
+The output from the component will be a list of matched *whole* rows (of type record). So, in our case, if one of inputs is (True, True, "myCar"), then the output would be a list consisting of a single recordList[(True, True, True)]. If you want to only get the _result columns_, then you need to transform the data afterwards with a different component.
+
+You can add as many input and result columns as you would like. The _Match condition_ is a fully functioning expression, so you can be quite flexible with it. You could omit some of the checks on the input rows. In our example, we could check for engine control only with *#engine = #ROW.Engine_starts*. It is also up to you which columns you define as input and which as output, you can treat them completely differently in two components that have the same table definition.
+
+[Download showcase fragment](showcases/decisionTableShowcase.json)
```

Comment:
Why fragment, not just tghe scenario?

Reply:
Idk, I just wrote a fragment for this...

Reply:
IMO it's better to have it as a scenario. 

But I see a much more important problem with the example. It's great to have it, but on the other hand, it's very dangerous because for sure we miss the moment when the scenario will be broken (because of reasons ... eg. JSON format changes). It'd be nice to have it tested. Eg. we could have a mechanism that takes all the example scenarios used in the docs and tries to import them (and checks if there are no errors). Without a mechanism like that, the example will be outdated (soon or later), and the users' feeling about the doc will be worse than without the example.

Reply:
We can have the scenarios imported on staging and scrape the website for correctness or have some check like you said. At the very least we can do "example scenario - designed for version x.y". Then the user will be prepared that it can be outdated and a slightly broken one will still be way better than nothing in my opinion. It doesn't have to be perfect.

Reply:
Ok, it's done